google-storage.yml

```diff
@@ -87,6 +87,16 @@ provision:
       Enables Uniform bucket-level access to a bucket.
       For more information about uniform bucket-level access, see https://cloud.google.com/storage/docs/uniform-bucket-level-access
     default: true
+  - field_name: default_kms_key_name
+    type: string
+    details: |
+      The `id` of a Cloud KMS key that will be used to encrypt objects inserted into this bucket.
+      Its default value is `empty` so a Google-managed encryption key will be used instead.
```

Comment:
Is the default `empty` (as a names constant), or `""` (empty string), or are they the same?

Reply:
Hey @blgm , thank you very much for reviewing the PR. Its default value is `""`. I see it can cause confusion, so I'll change it.

Thanks!